### ISSUE TITLE
Can't add caption on uploadPhoto

### ISSUE THREAD

**Issue by Latand** [LevPasha/Instagram-API-python#482]
Hey, I'm trying to send the image with caption, but image successfully uploads without it. There is no error or anything.

Am I doing something wrong?

Here is my code.
```

#!/usr/bin/env python
# -*- coding: utf-8 -*-
from InstagramAPI import InstagramAPI

captionText = """Test"""

ig = InstagramAPI("xxxx", "xxxx")
ig.login()


error_link = ig.LastJson.get("challenge")
if error_link:
    error_link = error_link.get("url")
    print(error_link)


try:
    ig.uploadPhoto("2.jpg", captionText)
except Exception as err:
    print(err)
```

**Comment by renobaron** [comment]
> Hey, I'm trying to send the image with caption, but image successfully uploads without it. There is no error or anything.
> 
> Am I doing something wrong?
> 
> Here is my code.
> 
> ```
> 
> #!/usr/bin/env python
> # -*- coding: utf-8 -*-
> from InstagramAPI import InstagramAPI
> 
> captionText = """Test"""
> 
> ig = InstagramAPI("xxxx", "xxxx")
> ig.login()
> 
> 
> error_link = ig.LastJson.get("challenge")
> if error_link:
>     error_link = error_link.get("url")
>     print(error_link)
> 
> 
> try:
>     ig.uploadPhoto("2.jpg", captionText)
> except Exception as err:
>     print(err)
> ```

Hi.
Try using 
`captionText = "Test"`

cheers

**Comment by Latand** [comment]
Tried different captions, just the first one 'Test' worked. After that every post is posted without a comment.
Then I tried 'Test' caption again - didn't show up!

**Comment by harishraman94** [comment]
Same. It worked the first time for me, but after that it did not work. The image uploads without the caption.

**Comment by renobaron** [comment]
I think Instagram will disallow posting with captions if you exceeds certain number of posts per hour.

**Comment by harishraman94** [comment]
No. I have hardly posted 2 photos with the same hour. It still posts without any caption. Help!

**Comment by deounix** [comment]
@harishraman94 ur account in shadow banned! u must wait 48 hour before u start post again

**Comment by Nachtjagdgeschwader** [comment]
Same problem: any caption won't show up.
```
InstagramAPI = InstagramAPI("xxx", "xxx")
InstagramAPI.login()
photo_path = 'squared.jpg'
mycap = 'test'
InstagramAPI.uploadPhoto(photo_path, caption=mycap)
```
The photo is uploaded and the caption is not.

**Comment by Nachtjagdgeschwader** [comment]
Now, I suspect that I'm also shadow banned. For those, who also can't use captions: try to edit a caption via official mobile app, try to comment other people posts. If you can't, then you're in my situation and probably is shadow banned.
I'll keep silence in Instagram for a couple of days and then retry commenting and posting.

**Comment by harishraman94** [comment]
@deounix My account is not shadow banned. The caption is working like 2 out of 5 times, and the other times it is not.

**Comment by deounix** [comment]
@harishraman94 Yeah I know. that's called shadow ban. I'm using this library with more than 200 accounts and I've about 30 account in shadow ban.

In my situation it's happen cuz bad proxy address.

What is bad proxy address ?
it's proxy some how Instagram know it's proxy server or it's using with some one before me with Instagram and get shadow banned.

U must stop posting new photos even the caption is working 2 from 5 times cuz ur account ll get banned!
U must wait at lest 48 hours.
